src/UI/Components/Diagram/actions.ts

```diff
@@ -465,6 +478,59 @@ export function appendEntity(
   return instanceAsTable;
 }
 
+/**
+ * Populates a graph with default required entities derived from a service model.
```

Comment:
isn't it an init method then?

Reply:
It's part of init, when I think about the init I think of creating all of JointJS elements required, and populating graph either by this function, or the other which goes through fetched instance and split it into shapes and append it to the canvas

Reply:
if it's only applied to initialize the graph, maybe a init-keyword in the naming would be nice. or a default keyword. Whichever you like best.

Reply:
changed to `defaultGraphPopulation()`